cookbooks/travis_build_environment/attributes/default.rb

```diff
@@ -268,9 +268,9 @@
 default['travis_build_environment']['mongodb']['service_enabled'] = false
 default['travis_build_environment']['mongodb']['keep_repo'] = false
 
-default['travis_build_environment']['shellcheck_url'] = 'https://storage.googleapis.com/shellcheck/shellcheck-v0.5.0.linux.x86_64.tar.xz'
-default['travis_build_environment']['shellcheck_version'] = '0.5.0'
-default['travis_build_environment']['shellcheck_checksum'] = '7d4c073a0342cf39bdb99c32b4749f1c022cf2cffdfb080c12c106aa9d341708'
+default['travis_build_environment']['shellcheck_url'] = 'https://storage.googleapis.com/shellcheck/shellcheck-v0.6.0.linux.x86_64.tar.xz'
+default['travis_build_environment']['shellcheck_version'] = '0.6.0'
+default['travis_build_environment']['shellcheck_checksum'] = 'ee51124ad2d9bb0ff2cc88a6aabd217d68bd34fb16ba0c79f580bb0335ce4f4f'
```

Comment:
Thanks!

I get a different sha256sum on the archive. Maybe it has been recompiled?

Reply:
Seems so, updated!